rootfs/docker-entrypoint.sh

```diff
@@ -80,29 +80,37 @@ if [ "$1" = 'postgres' ]; then
 		EOSQL
 		echo
 
+		gosu postgres pg_ctl -D "$PGDATA" -m fast -w stop
```

Comment:
not any more! The commands that *do* require postgres to be running are now run after the database has been booted and is running as seen further down below.